lib/dbservice_web/controllers/group_controller.ex

```diff
@@ -124,4 +107,39 @@ defmodule DbserviceWeb.GroupController do
       send_resp(conn, :no_content, "")
     end
   end
+
+  swagger_path :update_users do
+    post("/api/group/{groupId}/update-users")
+
+    parameters do
+      groupId(:path, :integer, "The id of the group", required: true)
+      body(:body, Schema.ref(:UserIds), "List of user ids to update", required: true)
+    end
+
+    response(200, "OK", Schema.ref(:GroupUsers))
+  end
+
+  def update_users(conn, %{"group_id" => group_id, "user_id" => user_id}) when is_list(user_id) do
```

Comment:
Repeating feedback. Please update at all places
```suggestion
  def update_users(conn, %{"group_id" => group_id, "user_ids" => user_ids}) when is_list(user_ids) do
```